tags/state/marko.json

```diff
@@ -0,0 +1,23 @@
+{
+  "<state>": {
+    "template": "./index.marko",
+    "transformer": "./transformer.js",
+    "@variable": {
+      "type": "expression",
+      "autocomplete": [
+        {
+          "description": "Variable to declare"
```

Comment:
I feel that this description could be better, maybe `The name of the state variable`